app/controllers/instruments_controller.rb

```diff
@@ -1,9 +1,35 @@
 class InstrumentsController < ApplicationController
+  # This ensures that users are logged in for actions that modify data
+  before_action :authenticate_user!, only: [:new, :create]
```

Comment:
I think in the lecture they taught that the best practice would be the white listing approach. So protect every route in the applications controller as a default and then we would not have to insert it here, but do `skip_before_action :authenticate_user!, only: %i[index home]`